File: Triatlon-Java/ChatModel/src/main/java/triathlon/model/Stage.java
package triathlon.model;

import java.io.Serializable;

public class Stage extends Entity<Integer> implements Serializable {
    private String name;

    public Stage(String name) {
        this.name = name;
    }

    public Stage() {}

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: Triatlon-CS/Model/Stage.cs
using System;

namespace Triatlon_CSharp.Domain
{
    [Serializable]
    public class Stage : Entity<int>
    {
        public Stage() {}
        
        public Stage(string name)
        {
            Name = name;
        }

        public String Name { get; set; }

        public override string ToString()
        {
            return string.Format("Id = {0} | {1}", Id, Name);
        }
    }
}